### ISSUE TITLE
关于 log_probs，问题

### ISSUE THREAD

**Issue by CurryX-001** [XueZeyue/DanceGRPO#43]
samples = {
        "timesteps": timesteps.detach().clone()[:, :-1],
        "latents": all_latents[
            :, :-1
        ][:, :-1],  # each entry is the latent before timestep t
        "next_latents": all_latents[
            :, 1:
        ][:, :-1],  # each entry is the latent after timestep t
        "log_probs": all_log_probs[:, :-1],
这个感觉有的问题，最后到 x0 的log_probs，好像这个是没有算最后的一个东西
